### ISSUE TITLE
frameRate cannot be 0 - only >0

### ISSUE THREAD

**Issue by SimonTate** [processing/p5.js#2014]
#### Nature of issue?

- [X ] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [X] Core
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [X ] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: 0.5.11
- Web browser and it's version: Safari Version 10.1.1 (12603.2.4)
- Operating System: macOS Sierra Version 10.12.5
- Steps to reproduce this:

```
function setup() {
  frameRate(30);
  textSize(20);
  textSize(30);
  textAlign(CENTER);
}
function draw() {
  background(200);
  text(frameCount, width/2, height/2);
}

function keyPressed() {
  if (key == '1'){
    frameRate(30); // resets frameCount
  }
  if (key == '2'){
    frameRate(0); // this line doesn't stop the frameCount
    frameRate(0.1); // this line does pause the count but why should it be used instead of 0? 
  }
}

```

**Comment by lmccart** [comment]
hm I think frameRate(0) is not supported. if you'd like to stop the looping altogether, you could use noLoop()

**Comment by SimonTate** [comment]
Ah yes, noLoop() and loop() are suitable for my application. I did however notice this:
```
p5.prototype.frameRate = function(fps) {
  if (typeof fps !== 'number' || fps <= 0) {
    return this._frameRate;
  } else {
    this._setProperty('_targetFrameRate', fps);
    this._runFrames();
    return this;
  }
};
```
within environment.js which seems to be doing the prevention of frameRate to be 0. I changed this to `fps < 0` in my own fork and it ran all grunt tests okay and worked for my application too. However if this is intentional then I'll go over to using noLoop() and loop(). 😄
